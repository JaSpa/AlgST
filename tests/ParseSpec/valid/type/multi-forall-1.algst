forall (a:TU) (b:TU). a
