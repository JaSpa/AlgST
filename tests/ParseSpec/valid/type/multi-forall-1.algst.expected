Type.Forall a:TU
|
`- Type.Forall b:TU
   |
   `- Type.Var a

